MarsRover/FileInput.cs

```diff
@@ -9,10 +9,10 @@ public string Read(string input)
             return File.ReadAllText(input);
         }
 
-        public string FileExists(string filePath)
+        public bool FileExists(string filePath)
         {
-            if (File.Exists(filePath)) return filePath;
-            return "File does not exist.";
+            if (File.Exists(filePath)) return true;

```

Comment:
you can just do

`return File.Exists(filePath)` as File.Exists returns a bool already. However, the filePath parameter is nullable (denoted by the question mark) so it's good practice to handle that too. https://docs.microsoft.com/en-us/dotnet/api/system.io.file.exists?view=net-5.0